CMakeLists.txt

```diff
@@ -172,6 +176,15 @@ if(UNIX AND NOT APPLE)
   endif()
   find_package(X11 COMPONENTS X11_xcb)
 endif()
+
+find_package(Qt6 REQUIRED COMPONENTS Protobuf)
+if(NOT Protobuf_FOUND)
```

Comment:
Remove: No need to use Protobuf directly.

Reply:
Done